CHANGELOG.md

```diff
@@ -1,10 +1,11 @@
 # Changelog
 
-This project uses [towncrier](https://towncrier.readthedocs.io/) and the
-changes for the upcoming release can be found in
+This project uses [towncrier](https://towncrier.readthedocs.io/) to generate changelogs.
+
+The changes for the upcoming release can be found in
 <https://github.com/ansys/grantami-jobqueue/tree/main/doc/changelog.d/>.
 
-<!-- towncrier release notes start -->
+[Published release notes](https://jobqueue.grantami.docs.pyansys.com/version/stable/changelog.html) can be found in the online documentation.]
```

Comment:
I assume this is from the standard template, but "Published release notes" seems like an odd way to describe the difference between md and rst formats. These release notes are still published, they're just published in a markdown document.

Maybe "Release notes for more recent releases can be found in the online documentation." and have the link on the "online documentation" text?